frameworks/kafka/tests/test_availability.py

```diff
@@ -0,0 +1,117 @@
+import datetime
+
+import subprocess
+#from dateutil.parser import *
+import pytest
+import os
+import re
+import time
+
+import sdk_cmd
+import sdk_install
+import sdk_plan
+import sdk_tasks
+import sdk_utils
+
+from tests.test_utils import *
+
+BROKERS_KILL_GRACE_PERIOD = int(os.environ.get('BROKER_KILL_GRACE_PERIOD', 30))
+EXPECTED_KAFKA_STARTUP_SECONDS = os.environ.get('KAFKA_EXPECTED_STARTUP', 30)
+EXPECTED_DCOS_STARTUP_SECONDS = os.environ.get('DCOS_EXPECTED_STARTUP', 30)
+STARTUP_POLL_DELAY_SECONDS = os.environ.get('STARTUP_LOG_POLL_DELAY', 2)
+
+def setup_module(module):
+    options = {
+        "brokers": {
+            "kill_grace_period": BROKER_KILL_GRACE_PERIOD
+        }
+    }
+
+    sdk_install.uninstall(SERVICE_NAME, PACKAGE_NAME)
+    sdk_utils.gc_frameworks()
```

Comment:
Please don't do this.  We lost logs in CI because of this.  Let the cluster's fill.  Mesos can GC.